lib/CMakeLists.txt

```diff
@@ -58,9 +58,9 @@ if (CMAKE_COMPILER_IS_GNUCXX)
     set_target_properties(libofx PROPERTIES CXX_VISIBILITY_PRESET hidden)
 endif (CMAKE_COMPILER_IS_GNUCXX)
 
-if (MSVC)
```

Comment:
actually I'm a bit confused now. Isn't a mingw/gcc build using a different library naming scheme compared to the MSVC build? I don't care that much, as I feel responsible only for the linux build, though.

And as a side note: If you feel able to add a windows build github action so that the successful build gets tested by the action, I would be all for this.

Reply:
Yes, I suppose it should be MSVC, after all. Fixed it.

Reply:
As for the GitHub action, that absolutely is something on my TODO list, although I never used GitHub pipelines before, so there's some learning curve involved.